templates/stamp/mysql-ubuntu-install.sh

```diff
@@ -160,8 +154,8 @@ install_mysql_server()
 
     package=$MYSQL_SERVER_PACKAGE_NAME
 
-    # todo: another conditional is required for sql5.6 on ubuntu16. 
-    if (( $(echo "$OS_VER < 16" |bc -l) )) && [ $mysqlServerPackageVersion == "5.7" ]
+    # todo: another conditional is required for sql5.6 on ubuntu16.
+    if (( $(echo "$OS_VER < 16" |bc -l) )) && [ $PACKAGE_VERSION == "5.7" ]
```

Comment:
fyi, this corrects a mistake i introduced. it thankfully didn't cause any undesired behavior because we were intending to use 5.6 anyway.